index.html

```diff
@@ -348,9 +348,21 @@ <h2>Transferable MediaStreamTrack</h2>
     <div>
       <p>A {{MediaStreamTrack}} is a <a data-cite="!HTML/#transferable-objects">transferable object</a>.
       This allows manipulating real-time media outside the context it was requested or created in, for instance in workers or third-party iframes.</p>
-      <p>To preserve the existing privacy and security infrastructure, in particular for capture tracks, we need proper track and source lifetime management.
-      A future PR will properly specify this management.</p>
-      <p>To preserve the existing privacy and security infrastructure, in particular for capture tracks,
+      <p>To preserve the existing privacy and security infrastructure, in particular for capture tracks, the track source lifetime management remains tied to the context that created it.
+      The transfer algorithm MUST ensure the following behaviors:</p>
+      <p>
+        <ol>
+          <li><p>The context that created a track named <var>originalTrack</var> remains in control of the source that provides media, even when <var>originalTrack</var> is transferred into <var>transferredTrack</var>.
+          <li>
```

Comment:
Suggest using "originalSource" or "orignalContext" as a varilable rather than "the context of originalTrack"

Reply:
I introduced trackSource and originalContext